KinSDK/KinSDK/Core/blockchain/types/TransactionFactory.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  TransactionFactory.swift
+//  KinSDK
+//
+//  Created by Corey Werner on 17/07/2019.
+//  Copyright © 2019 Kin Foundation. All rights reserved.
+//
+
+import Foundation
+
+class TransactionFactory {
```

Comment:
how `TransactionFactory` is used by the consumer?

Reply:
https://github.com/kinecosystem/kin-sdk-ios/pull/49/files#diff-91a576d1a878f1380a7a95a46b4e4e50R365
https://github.com/kinecosystem/kin-sdk-ios/pull/49/files#diff-91a576d1a878f1380a7a95a46b4e4e50R338